strawberry/types/fields/resolver.py

```diff
@@ -1,10 +1,10 @@
-from __future__ import annotations
+from __future__ import annotations as _
```

Comment:
is this done on purpose? :)

Reply:
Yeah, I defined a method-scope variable called `annotations` and my linter was complaining about symbol shadowing